src/share-buttons.jsx

```diff
@@ -20,6 +20,7 @@ export default class ShareButton extends Component {
     windowWidth: PropTypes.number,
     windowHeight: PropTypes.number,
     beforeOnClick: PropTypes.func,
+    closeCallback: PropTypes.func,
```

Comment:
To make it more explicit, I think this function should be named `onShareWindowClose`.